torchtune/models/phi3/_component_builders.py

```diff
@@ -252,7 +254,7 @@ def lora_phi3_self_attention(
     num_kv_heads: int,
     max_seq_len: int,
     attn_dropout: float = 0.0,
-    rope_base: float = 500000.0,
+    rope_base: int = 10_000,
```

Comment:
I like this format since its more readable than 10000. Can we revert back to this format above as well?